tests/test_column_state.py

```diff
@@ -152,4 +231,15 @@ def display_column_state(col_state):
     until(lambda: json.dumps(testState) in dash_duo.find_element('#reset-column-state-grid-pre').text, timeout=3)
 
     dash_duo.find_element('#load-column-state-button').click()
+    until(lambda: json.dumps(colState) in dash_duo.find_element('#reset-column-state-grid-pre').text, timeout=3)
+
+    (
+        ActionChains(dash_duo.driver)
+            .move_to_element(dash_duo.find_element('#load-column-defs'))
+            .click()
+    ).perform()
```

Comment:
Is there some reason it doesn't work to just click the button? If it's scrolled off-screen maybe we can just put it on a new line or shorten the labels or something?
```suggestion
    dash_duo.find_element('#load-column-defs').click()
```

Reply:
Clicking it is too fast for some reason. 

There has to be a slight pause between clicks or it doesn't work.

Reply:
ok - maybe just add a little `time.sleep` then, with a comment about why?

Reply:
I was trying to avoid that. Haha. 

But yes, a time.sleep works.

Reply:
haha yeah `time.sleep` is generally not great, but if there's a real reason we need a delay rather than waiting for a specific event, it's better to do that explicitly rather than something implicit like this.